src/Navigation/Header/styled.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+import { ReactComponent as LogoWebsite } from "./logo.svg"
+
+export const StyledHeader = styled.header`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 12px;
+    padding-top: 5px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px){
+        gap: 6px;
+    };
+    
+    @media(max-width: ${({ theme }) => theme.breakpoints.mobile}px){    
+        justify-content: space-evenly;
+        gap: 8px;
+    };
+`;
+
```

Comment:
If I remeber correctly, as a rule, semicolons are not used after CSS declarations.

If you are using the Prettier extension maybe you have that in setting that on Alt+Shft+F it addes ";" everywhere 😉